sdk/confidentialledger/confidential-ledger-rest/test-resources.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ledgerName": {
+      "type": "string",
+      "metadata": {
+        "defaultValue": "0000-test-js",
+        "description": "Ledger Name"
+      },
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "principalId": {
+      "type": "string",
+      "defaultValue": "ec667af1-0642-45f0-be8a-b76758a35dde",
+      "metadata": {
+        "description": "Oid of the user"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47"
+    },
+    "clientId": {
+      "type": "string",
+      "defaultValue": "11117bee-56a6-4430-9988-b8788a90c2e9"
+    },
+    "clientSecret": {
+      "type": "string",
+      "defaultValue": "test"
+    }
+  },
+  "functions": [],
+  "variables": {},
+  "resources": [
+    {
+      "name": "[parameters('ledgerName')]",
+      "type": "Microsoft.ConfidentialLedger/ledgers",
+      "ledgerUri": "[concat('https://', parameters('ledgerName'), '.confidential-ledger.azure.com')]",
+      "apiVersion": "2020-12-01-preview",
+      "location": "[parameters('location')]",
+      "properties": {
+        "ledgerType": "Public",
+        "aadBasedSecurityPrincipals": [
+          {
+            "principalId": "[parameters('principalId')]",
+            "tenantId": "[parameters('tenantId')]",
+            "ledgerRoleName": "Administrator"
+          },
+          {
+            "principalId": "[parameters('clientId')]",
+            "ledgerRoleName": "Administrator"
+          }
+        ]
+      }
+    }
+  ],
+  "outputs": {
+    "LEDGER_NAME": {
+      "type": "string",
+      "value": "[parameters('ledgerName')]"
+    },
+    "LEDGER_URI": {
+      "type": "string",
+      "value": "[resources[0]('ledgerUri')]"
+    },
+    "USER_AAD": {
+      "type": "string",
+      "value": "[parameters('principalId')]"
+    },
+    "TENANT_ID": {
+      "type": "string",
+      "value": "[parameters('tenantId')]"
+    },
+    "CLIENT_ID": {
+      "type": "string",
+      "value": "[parameters('clientId')]"
+    },
+    "CLIENT_SECRET": {
+      "type": "string",
+      "value": "[parameters('clientSecret')]"
```

Comment:
These values should not be included in the ARM template outputs. Our deployment scripts will already set them in the environment via `CONFIDENTIALLEDGER_CLIENT_SECRET`, etc.